testing/file_test/testdata/alternating_files.carbon

```diff
@@ -0,0 +1,24 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+// AUTOUPDATE
+// CHECK:STDOUT: 3 args: `default_args`, `a.carbon`, `b.carbon`
+// CHECK:STDOUT: unattached message 1
+// CHECK:STDERR: unattached message 1
```

Comment:
The relative order of `STDOUT` and `STDERR` seems surprising to me here? But also maybe not as important compared to other changes...

Reply:
FWIW my leaning would be to swap this if it's easy, since these tests aren't representative. Typically in the toolchain I'd be expecting stderr to be prioritized, although diagnostics would be associating lines with stderr so it'd be largely moot (and thus not worth too much time).

Reply:
Done.